diff -uar ./TensorFlowServerScriptFormatter.java.patch ./ignite/modules/tensorflow/src/main/java/org/apache/ignite/tensorflow/cluster/tfrunning/TensorFlowServerScriptFormatter.java
--- ./ignite/modules/tensorflow/src/main/java/org/apache/ignite/tensorflow/cluster/tfrunning/TensorFlowServerScriptFormatter.java	2019-05-30 12:22:53.000000000 -0500
+++ ./ignite/modules/tensorflow/src/main/java/org/apache/ignite/tensorflow/cluster/tfrunning/TensorFlowServerScriptFormatter.java       2019-05-30 14:17:12.086780814 -0500
@@ -67,8 +67,8 @@
 
         builder.append("import tensorflow as tf").append('\n');
         builder.append("to_import_contrib_ops = tf.contrib.resampler").append("\n");
-        builder.append("from tensorflow.contrib.ignite import IgniteDataset").append("\n");
-        builder.append("import tensorflow.contrib.igfs.python.ops.igfs_ops").append("\n");
+        builder.append("from tensorflow_io.ignite import IgniteDataset").append("\n");
+        builder.append("import tensorflow_io.ignite.python.ops.igfs_ops").append("\n");
 
         builder.append("print('job:%s task:%d' % ('")
             .append(srv.getJobName())
